pages/about.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Code 4 Easy</title>
+    <link rel="stylesheet" href="/../assets/css/about.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;500;600;700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css"
+        crossorigin="anonymous" />
+</head>
+
+<body>
+    <main>
+        <header>
+            <div class="desktop-nav">
+                <nav class="container">
+                    <a class="logo" href="/">Code 4 Easy</a>
+                    <ul class="list">
+                        <li><a href="/">Home</a></li>
+                        <li><a href="/pages/about.html">About us</a></li>
+                        <li><a href="#">Course</a></li>
+                        <li><a href="#">Join</a></li>
+                        <li><a href="#">Sponsor</a></li>
+                        <li><a href="#">News</a></li>
+                    </ul>
+                </nav>
+            </div>
+            <div class="about">
+                <div>
+                    <div class="mobile-nav" id="mobile-nav">
+                        <div class="menu">
+                            <div></div>
+                            <div></div>
+                            <div></div>
+                        </div>
+                    </div>
+                </div>
+                <div class="headline container">
+                    <p class="title-sub">"Hello! Developers"</p>
+                    <h1 class="font-cocogoose title">CODE 4 EASY DEV BOOTCAMP 2022</h1>
+                    <p class="caption">
+                        Code 4 Easy Bootcamp takes all its students from knowing nothing about
+                        programming or web programming or anything related to independent
+                        software developer within 5 months of intensive learning and coding.
+                    </p>
+                    <p class="contact-info">Please contact us per Email for any further questions about Code 4 Easy
+                        Bootcamp</p>
+                    <a href="email:ishmaelkargbo5@outlook.com">Ishmaelkargbo5@gmail.com</a>
+                </div>
+            </div>
+        </header>
+        <section class="past container">
```

Comment:
The about page, is missing a section.

- [ ] In order to respect the design, kindly, build the missing section;

Given design:
![image](https://user-images.githubusercontent.com/83596135/163476514-2809029f-9a40-4361-84f0-c58f2bbf1e9a.png)